runtime/gc_modron_startup/mminit.cpp

```diff
@@ -350,14 +350,17 @@ j9gc_initialize_heap(J9JavaVM *vm, IDATA *memoryParameterTable, UDATA heapBytesR
 			const char* qualifier = NULL;
 			qualifiedSize(&size, &qualifier);
 
-			char *format = (char *)j9nls_lookup_message(J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE, J9NLS_GC_FAILED_TO_INSTANTIATE_LOW_MEMORY_RESERVE_SIZE_REQUESTED, "Failed to instantiate compressed references metadata.  %zu%s requested");
+			const char *format = j9nls_lookup_message(
+				J9NLS_DO_NOT_PRINT_MESSAGE_TAG | J9NLS_DO_NOT_APPEND_NEWLINE,
+				J9NLS_GC_FAILED_TO_INSTANTIATE_LOW_MEMORY_RESERVE_SIZE_REQUESTED,
+				"Failed to instantiate compressed references metadata.  %zu%s requested");
```

Comment:
Please update the string literal so it is consistent with `j9modron.nls`:
```c
				"Failed to instantiate compressed references metadata; %zu%s requested");
```
Please do likewise for the literal on line 378:
```c
				"Failed to instantiate heap; %zu%s requested");
```